README.md

```diff
@@ -56,4 +56,7 @@ By default, all scopes are allowed.
 ```scala
 ThisBuild / conventionalCommits / scopes := Seq() // Allow all scopes
 ThisBuild / conventionalCommits / scopes := Seq("scope1", "scope2") // Allow only scope1 and scope2
-```
\ No newline at end of file
+```
+
+You can also configure the success and failure messages displayed when committing, respectively by changing the
+`ThisBuild / conventionalCommits / successMessage` and `ThisBuild / conventionalCommits / failureMessage` keys.
```

Comment:
I'd show the different options and consequences:

You can also configure the success and failure messages displayed when committing, respectively by changing the
`ThisBuild / conventionalCommits / successMessage` and `ThisBuild / conventionalCommits / failureMessage` keys:

```scala
ThisBuild / conventionalCommits / successMessage = None // Won't display a success message
ThisBuild / conventionalCommits / successMessage = Some("custom message") // Shows custom message
```